internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/integration/utils/VeniceClusterWrapper.java

```diff
@@ -165,6 +168,12 @@ public class VeniceClusterWrapper extends ProcessWrapper {
     this.nettyServerToGrpcAddress = nettyServerToGrpcAddress;
   }
 
+  public Properties getPubSubClientProperties() {
+    Properties props = new Properties();
```

Comment:
Can we use a Properties object for pubBrokerDetails to avoid recreating it on every call? Alternatively, we could just return an unmodifiable map.